ui/app/styles/helper-classes/typography.scss

```diff
@@ -104,3 +104,7 @@
     color: inherit;
   }
 }
+
+.text-disabled {
+  opacity: 0.6;
+}
```

Comment:
I wonder if this class could be more generalized like `.opacity-060` or something along those lines? I'm just thinking if you wanted to use this on an element without text to reduce the opacity it might not be clear what the class is doing.

Reply:
I like it. I'll amend.